### DISCUSSION TITLE
How to create a custom field?

### DISCUSSION THREAD

**Discussion by elfeffe** [filamentphp/filament#125] [Help]
I have created a livewire block editor that basically returns an array of blocks to a master component, so you can save the data.
I need to know how to create a field that can be the master component which extends my builder, so I can use the blocks builder with filament.
Is there any tutorial, or can you guide me a bit?

**Comment by danharrin** [comment]
Hey! 👋

I'm pleased to announce that a block editor is now a feature of [Filament Form Builder v2](https://filamentadmin.com/docs/forms). When Filament Admin Panel v2 is released in Winter 2021, it will be available there. For more information, please see #495.

In the meantime, feel free to try out the feature in the Form Builder and let me know if it fits your needs! It only takes a minute to install.

If you have any questions, let me know in a reply!

Dan

**Comment by elfeffe** [comment]
And is there any way to preview the block builder?
In any case, it will be great if we can add our fields.
My builder is very specific, and I would want to use it.

**Reply by elfeffe** [reply]
Ok, I saw the docs about custom fields.
